addon/components/frost-list-item.js

```diff
@@ -1,22 +1,48 @@
-/* global $ */
 import Ember from 'ember'
-import _ from 'lodash/lodash'
-import FrostList from '../frost-list/component'
+const {Component, $} = Ember
+import computed, {readOnly} from 'ember-computed-decorators'
+import {PropTypes} from 'ember-prop-types'
+import _ from 'lodash'
+import FrostList from './frost-list'
 
-export default Ember.Component.extend({
-  classNameBindings: ['isSelected', 'frost-list-item'],
+export default Component.extend({
 
-  initContext: Ember.on('init', function () {
-    this.set('_frostList', this.nearestOfType(FrostList))
-  }),
+  // == Dependencies ==========================================================
+
+  // == Properties ============================================================
+  classNames: ['frost-list-item'],
+  classNameBindings: [
+    'isSelected',
+    'showDetail:is-expanded:is-collapsed'
```

Comment:
If you use `isExpanded` then the `:` specifications aren't necessary (will be either `is-expanded` or nothing)